src/cswinrt/code_writers.h

```diff
@@ -498,6 +498,18 @@ namespace cswinrt
         w.write("default(%)", bind<write_projection_type>(semantics));
     }
 
+    void write_event_out_defaults(writer& w, method_signature const& methodSig)
+    {
+        for (auto&& param : methodSig.params())
+        {
+            if (get_param_category(param) == param_category::out || get_param_category(param) == param_category::receive_array)
+            {
+                w.write("\n% = default(%);", bind<write_parameter_name>(param), bind<write_projection_type>(get_type_semantics(param.second->Type())));
+                break;
```

Comment:
Can there be more than 1 out parameter that we need to do for each rather than breaking?